samples/matter/common/src/bridge/bridged_device_data_provider.h

```diff
@@ -22,6 +22,7 @@ class BridgedDeviceDataProvider {
 	virtual void NotifyUpdateState(chip::ClusterId clusterId, chip::AttributeId attributeId, void *data,
 				       size_t dataSize) = 0;
 	virtual CHIP_ERROR UpdateState(chip::ClusterId clusterId, chip::AttributeId attributeId, uint8_t *buffer) = 0;
+	virtual CHIP_ERROR SetValue(const void *newValue) { return CHIP_NO_ERROR; };
```

Comment:
Hmm indeed... would having specific setters on each concrete device provider interface work for you (SetTemperature, SetHumidity etc.)? We need to distinguish the device type in the shell module anyway, so that would just imply few lines of code more.

Reply:
Yes, that could also work, but the shell could just call `NotifyUpdateState` directly and it still would work, right? I mean `mTemperature` wouldn't be updated, but this doesn't seem to change the behavior (`mTemperature` member could be used only for the automatic provider variant).

Reply:
Done